libs/form/src/next/upload/stories/vl-upload.stories-doc.mdx

```diff
@@ -0,0 +1,93 @@
+import { ArgTypes, Canvas } from '@storybook/addon-docs';
+import * as VlUploadStories from './vl-upload.stories';
+import FormControlPublicMethods from '../../form-control/stories/form-control.public-methods-doc.mdx'
+
+# Upload - next
+
+Gebruik de `upload-next`-component om één of meerdere bestanden te selecteren of te slepen naar het upload-veld. De gebruiker
+kan alternatief ook één of meerdere bestanden uploaden door op de link in het upload-veld te klikken en de bestanden
```

Comment:
Dit is een zin die vanonder vermeld kan worden, de tekst vanboven in het component liefst zo beknopt mogelijk houden.